storybook/tailwind.css

```diff
@@ -28,7 +26,7 @@ pre code {
   }
 
   .sbdocs-a {
-    color: $color-blue-400;
+    color: --color-blue-400;
```

Comment:
Might be a bit of a noob question, b/c I never really worked with CSS variables, but shouldn't that be `var(--color-blue-400)`?